### PR TITLE
PP-6285 Add cookie banner and analytics

### PR DESCRIPTION
**Pull request by kbottla**
## WHAT
- Added cookie banner, analytics and relevant tests
- Sets common cookie `gov_pay_cookie_policy` for analytics for consent choice
- Added new dev dependency `jsdom` for simulating DOM/window for testing cookie-banner/analytics functions

### CODE REVIEW

File: app/assets/sass/components/cookie-message.scss [hunk 1]
```diff
@@ -1,19 +1,112 @@
-.cookie-banner {
-  box-sizing: border-box;
```
Comment (iqbalgds) [inline app/assets/sass/components/cookie-message.scss:2]: Will the old banner work for now as well?

File: app/browsered/cookie-banner.js [hunk 2]
```diff
@@ -0,0 +1,265 @@
+'use strict'
+
+const analytics = require('./analytics')
```
Comment (iqbalgds) [inline app/browsered/cookie-banner.js:3]: Can we split the cookie functions out?
And require it in?

Comment (kbottla) [reply app/browsered/cookie-banner.js:3]: Added new file for cookie functions

File: app/assets/sass/components/cookie-message.scss (hunk 1 above)
Comment (kbottla) [reply app/assets/sass/components/cookie-message.scss:2]: we don't use this class any more

**Pull request merged**